tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/configreader/ConfigReader.java

```diff
@@ -212,16 +212,28 @@ private void populateIntermediateObject(Iterator<Attribute> attributes, String m
 
             final Object finalObject = object;
             attributes.forEachRemaining(attribute -> {
+                String property = attributeMapper.getmAttributeMapper().get(getAttributeName(attribute));
                 try {
-                    String property = attributeMapper.getmAttributeMapper().get(getAttributeName(attribute));
                     if (property != null) {
                         Field field = mClass.getDeclaredField(property);
                         field.setAccessible(true);
                         field.set(finalObject, attribute.getValue());
                     }
                 } catch (NoSuchFieldException e) {
-                    logger.warn(
-                            " NoSuchFieldException : There can be attributes in mule xml that is not " + "mapped " + e);
+                    //If the field cannot be found in the class check whether its available in the super class
+                    Class<?> superClass = mClass.getSuperclass();
+                    Field field = null;
+                    try {
+                        if (property != null && superClass != null) {
+                            field = superClass.getDeclaredField(property);
+                            field.setAccessible(true);
+                            field.set(finalObject, attribute.getValue());
+                        }
+                    } catch (NoSuchFieldException ex) {
+                        logger.warn(" NoSuchFieldException ", ex);
```

Comment:
Any reason why this is a warn log instead of an error log ?

Reply:
There can be attributes in mule xml that is not mapped. so it's not exactly an error. Just a warning.